app/controllers/api/events_controller.rb

```diff
@@ -0,0 +1,46 @@
+class API::EventsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_filter :cors_preflight_check
```

Comment:
@fuzzyalej I was just toying with it but I decided to remove it in the last commit. http://stackoverflow.com/questions/17858178/allow-anything-through-cors-policy